src/scroll-timeline-base.js

```diff
@@ -89,9 +115,14 @@ export function installScrollOffsetExtension(parseFunction, evaluateFunction) {
  * @returns {number}
  */
 export function calculateMaxScrollOffset(scrollSource, orientation) {
-  // TODO: Support other writing directions.
-  if (orientation === "block") orientation = "vertical";
-  else if (orientation === "inline") orientation = "horizontal";
+  // Only one horizontal writing mode: horizontal-tb.  All other writing modes
+  // flow vertically.
+  const horizontalWritingMode =
+    getComputedStyle(this.scrollSource).writingMode == 'horizontal-tb';
+  if (orientation === "block")
+    orientation = horizontalWritingMode ? "vertical" : "horizontal";
+  else if (orientation === "inline")
+    orientation = verticalWritingModel ? "vertical" : "horizontal";
```

Comment:
verticalWritingModel is undefined. This probably points to a lack of tests covering this.

Reply:
Oops.  Fixed.  Yes we have a hole in the test coverage.